openai_functools/functions_orchestrator.py

```diff
@@ -221,3 +228,26 @@ def create_function_descriptions(
             ]
         )
         return [spec.parameters for spec in specs]
+
+    def create_tools_descriptions(
+        self, selected_functions: Optional[List[str]] = None
+    ) -> List[Dict[str, Any]]:
+        """
+        Creates descriptions for the selected functions.
```

Comment:
Open question: the description of this function is very similar to create_function_descriptions. Can we assume that the user knows the difference between function descriptions and tool descriptions? Should we make it explicit in the docstring?

Reply:
Made the docstrings more explicit